passthrough/usecase/passthrough_usecase.go

```diff
@@ -238,6 +245,18 @@ func (p *passthroughUseCase) GetPortfolioAssets(ctx context.Context, address str
 		return unclaimedCoins, finalErr
 	}
 
+	getLimitOrderCoins := func(ctx context.Context, address string) (sdk.Coins, error) {
+		orders, _, err := p.orderbookUseCase.GetActiveOrders(ctx, address)
+		var limitOrdersCoins sdk.Coins
+		for _, order := range orders {
+			limitOrdersCoins = limitOrdersCoins.Add(sdk.NewCoin(
```

Comment:
Please switch to this for performance reasons. Coin constructor has extra overhead

```suggestion
			limitOrdersCoins = limitOrdersCoins.Add(sdk.Coin{
```